app/components/countdown_alert_component.rb

```diff
@@ -1,9 +1,11 @@
 # frozen_string_literal: true
 
 class CountdownAlertComponent < BaseComponent
-  attr_reader :show_at_remaining, :alert_options, :countdown_options, :tag_options
+  attr_reader :show_at_remaining, :screen_reader_frequency, :alert_options, :countdown_options,
+              :tag_options
 
   def initialize(
+    screen_reader_frequency: nil,
```

Comment:
Rather than adding a new option here, did you consider using the approach that we use in the session timeout modal? Namely, having two separate countdowns, one of which is used for assistive technology announces at a slower interval:

https://github.com/18F/identity-idp/blob/2b6458b9ebfa7f7b72435657c75af520b4b2d03f/app/views/session_timeout/_warning.html.erb#L10-L38

Reply:
I thought that I was going in this direction, but I am pivoting because I originally thought it was to use the one element.